modules/tools/kube-prometheus-stack/rules.tf

```diff
@@ -1,99 +0,0 @@
-resource "kubernetes_manifest" "prometheus_rule_kube_cluster-at-max-nodes" {
```

Comment:
There's another file in this module, `service-monitors.tf`, that's entirely commented out, but it also uses the `kubernetes_manifest` resource. That could probably be removed as well.

Reply:
good catch @alexashley, I moved those into the helm chart also and added a feature flag since we want to add those after we update k8s + harbor